README.md

```diff
@@ -10,4 +10,24 @@ Content from this repository is available in [Ansible Galaxy](https://galaxy.ans
 
 ```
 ansible-galaxy install redhat-cop.jcliff
-```
\ No newline at end of file
+```
+
+## Installing Wildfly using Ansible
```

Comment:
@rpelisse Do we want to add a prerequisites section to ensure the user has both ansible and jcliff installed?

Reply:
Installing JCliff is done by the second playbook, so no need. We could add a comment about installing Ansible, but it is an ansible role we are shipping, if the user got here, he must know that he needs to install Ansible, mustn't he?

Reply:
@rpelisse the issue with the playbook is that it is not platform independent. I cannot run it on my OSX machine. This is the issue that I am working to resolve with HomeBrew, but we first need to provide a more formal release process for jcliff that can be leveraged by all platforms

Reply:
Right, sorry, I keep forgetting about the non-yum world. Then, you are right, we need to indicate in the read me that people needs to install JCliff (using Maven and setting up everything manually) as a prerequisite. I'll update the README after you merged this PR, so that I don't have to merge/squash or whatev

Reply:
How about downloading the binary from github? I can add the binaries to the release.

Reply:
@bserdar that would be amazing!

Reply:
Added the tgz. Let me know if you need more, and I can attach them.

https://github.com/bserdar/jcliff/releases/tag/v2.12.0

Reply:
@bserdar great! I'll do some testing over the weekend